lib/IRGen/IRGenFunction.cpp

```diff
@@ -180,13 +180,9 @@ llvm::Value *IRGenFunction::emitVerifyEndOfLifetimeCall(llvm::Value *object,
 void IRGenFunction::emitAllocBoxCall(llvm::Value *typeMetadata,
                                       llvm::Value *&box,
                                       llvm::Value *&valueAddress) {
-  auto attrs = llvm::AttributeList::get(IGM.getLLVMContext(),
-                                        llvm::AttributeList::FunctionIndex,
-                                        llvm::Attribute::NoUnwind);
-
   llvm::CallInst *call =
     Builder.CreateCall(IGM.getAllocBoxFn(), typeMetadata);
-  call->setAttributes(attrs);
+  call->addFnAttr(llvm::Attribute::NoUnwind);
```

Comment:
@jckarter was this specifically replacing all attributes added in `CreateCall` or is using `add` okay (there's a few other cases like this)? The LLVM commits mention possibly hiding `AttributeList::AttrIndex` in the future so I tried to use the new `add(Fn|Param|Ret)*` methods where possible. There's just a single `FirstArgIndex` remaining, used in a `replace*` method that doesn't have a `(Fn|Param|Ret)` equivalent at the moment.

If `setAttributes` is used here on purpose I can just change this to `llvm::AttributeList().addFnAttr...` instead.